docs/release-notes/2.41.1.md

```diff
@@ -0,0 +1,38 @@
+
+# Percona Monitoring and Management 2.41.1
+
+| **Release date** | Jan 24, 2024                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations—regardless of whether your databases are located on-premises or in the cloud.
+
+## Release Highlights
+
+### Improved support for Arbiter nodes
+
+We've made it easier to add and monitor MongoDB Arbiter nodes in PMM, as this no longer requires workarounds for cases where authorization is enabled on the MongoDB replica set. Additionally, Arbiter nodes are now displayed correctly on the dashboards:
+
+![!](../_images/Arbiter.png)
+
+### New metric for sharded MongoDB insights
+
+We've added a new metric to our MongoDB exporters: `mongodb_shards_collection_chunks_count`. This metric is available for sharded Mongo instances, offering insights into the distribution of collection chunks across different shards.
+
+Explore and leverage this new metric to enhance your understanding of your sharded MongoDB setup.
+
```

Comment:
```suggestion

### New Experimental MongoDB Instance Summary dashboard
We have introduced a new MongoDB Instance summary in this release that offers PMM users a simplified view of the most critical MongoDB metrics. You can locate it in the "Experimental" folder of your PMM installation. We request you to verify it against your MongoDB instance and share your feedback on our forum.

```

Reply:
@catalinaadam  pls 👁️  this is for the [PMM-12750](https://perconadev.atlassian.net/browse/PMM-12750)